### PR TITLE
Add documentation about RADIUS configuration

### LINKED ISSUE #2448: Add RADIUS config docs
A documentation for the RADIUS server, that are configured in the system config in the webui is missing.
This is on the same level as the smtpserver_config.

This documentation needs to point out, that these are RADIUS server to which privacyIDEA forwards authentication requests for RADIUS tokens and for passthru-policies.

### PR DESCRIPTION
**Pull request by cornelinux**
This is to avoid the misunderstanding, that
the admin could configure a RADIUS server
that would receive a request.

Closes #2448

### CODE REVIEW

File: doc/configuration/radiusserver_config.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---------------------------
+
+.. index:: RADIUS server
+
+At *config->system->RADIUS servers* the administrator
+can configure to which RADIUS servers privacyIDEA can forward authentication requests.
+These RADIUS servers can be used with :ref:`RADIUS tokens <radius_token>`
+and in the :ref:`Passthru Policy <passthru_policy>`.
+
+.. note:: This is ment for outgoing RADIUS requests, not for incoming RADIUS requests!
```
[the remark this answers is not in the document]
Comment (plettich) [reply doc/configuration/radiusserver_config.rst:13]: please fix this typo

**Review by plettich — CHANGES_REQUESTED** [review]
looks good, please fix typo

File: doc/configuration/radiusserver_config.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (cornelinux) [reply doc/configuration/radiusserver_config.rst]: done in 4a2c90b3

**Review by plettich — APPROVED** [review]
Looks good

**Pull request merged**